addons/base_import/static/src/import_action/import_action.js

```diff
@@ -50,6 +51,28 @@ export class ImportAction extends Component {
             previewError: "",
         });
 
+        this.uploadFiles = useFileUploader();
+        useDropzone(useRef("root"), async event => {
```

Comment:
Don't really understand how that turns only the no-content-helper into a drop zone, seems to use the whole component.

This allows re-dropping a file even when you're importing, which is cool I guess, but not really implied in commit message.

Reply:
The no-content helper element is too small and I think it's not where we should set the dropzone. Ideally, I would like to set the dropzone on the view gray zone but it involves more changes because the view gray zone is defined inside the generic `Layout` component.

Maybe, I can define a variant of the `Layout` component, plug the hook I want and pass a callback function `onDrop` to upload the file but I feel like it was a bit overkill. So, I just put it there on the root node of the view.

**Do you prefer having the dropzone on the view gray zone?** I can change that if you prefer :smiley:

If it's okay, I will update the commit message.

Reply:
I think it's fine to go for this, kind of cool that you can re-upload something multiple times by drag & dropping IMO.

So just changing the commit message should be sufficient 🙂